--- unit_tests/CMakeLists.txt       2020-02-22 02:31:34.726330772 -0600
+++ unit_tests/CMakeLists.txt   2020-02-22 02:30:33.905294797 -0600
@@ -36,7 +36,7 @@                                                                                                                                                     
 endfunction()                                                                                                                                                        
                                                                                                                                                                      
 ADD_SUPPORTED_TEST_PRODUCT_FAMILIES_DEFINITION()                                                                                                                     
-link_libraries(${ASAN_LIBS} ${TSAN_LIBS})                                                                                                                            
+link_libraries(${ASAN_LIBS} ${TSAN_LIBS} -lrt)
 
 add_custom_target(unit_tests)
 add_custom_target(prepare_test_kernels)
